docs/developers/devmining-reqs.md

```diff
@@ -27,7 +27,7 @@ If your contract's price identifier is not supported by the existing price feeds
 To calculate the correct number of rewards, it must be possible to determine the value of each synthetic token in a common base currency. Each synthetic token's value is converted to USD to determine the total value minted (TVM) by each contract. These TVMs are then used to determine how rewards should be allocated.
 
 To perform this process, each synthetic token must have an available price feed in USD. Before receiving rewards, you must ensure that one of the following is available.
```

Comment:
Where even is this page? I'm hoping it is no longer on the docs site, but either way we should just remove this entire page from the repo.